packages/website/components/MdxRenderer.tsx

```diff
@@ -41,12 +42,15 @@ const components = {
       <Table {...props} />
     </Box>
   ),
-  thead: (props) => <TableHead {...props} />,
-  tbody: (props) => <TableBody {...props} />,
-  tr: (props) => <TableRow {...props} />,
-  th: (props) => <TableCell style={{ textAlign: 'left' }} {...props} />,
-  td: (props) => <TableCell {...props} />,
+  thead: (props) => <Table.Head {...props} />,
+  tbody: (props) => <Table.Body {...props} />,
+  tr: (props) => <Table.Row {...props} />,
+  th: (props) => <Table.Cell style={{ textAlign: 'left' }} {...props} />,
+  td: (props) => <Table.Cell {...props} />,
+  ...f36Components,
   ...MdxComponents,
+  Props,
+  PropsHeading,
```

Comment:
We already spreading `MdxComponents` with `Props` and `PropsHeading`

Reply:
Oops!